tools/mFold.md

```diff
@@ -0,0 +1,131 @@
+# mFold
+
+written by: Kevin Yuan
+
+[{estimated time} minute] This tool is for tutorial 11. mFold is one of the earliest tools for computational molecular biology, operating since 1995. It is primarily used for prediction of nucleic acid folding, which can be valuable in evaluating and comparing the structure of viruses.
+
+**Tutorial Objective**: We will use mFold to generate several possible structures of a truncated narnavirus.
+
+## Input / Prerequisites
+
+-   [Tool Weblink](http://www.unafold.org//)
+
+-   Web browser
+
+-   A DNA or RNA sequence \< 2400 nucleic acids
+
+-   Example Data: AGACGGATGTGATAGATAAACTATCACTAGCTGATTATGGCACTTTAAAGCGCCTAATCA
```

Comment:
Put this nucleotide sequence into a "Code Block" so it's visually separate from the other text. Use the "`" character three times like so

```
This is in a code block. To put words into a code block you use three ` ` `
```